docs/source/user-guide/vision/VisionData.rst

```diff
@@ -0,0 +1,215 @@
+.. _vision_data_class:
+
+=====================
+The Vision Data Class
+=====================
+The :class:`VisionData <deepchecks.vision>` data class is the deepchecks base class for
+storing your data for a vision task. It is essentially a wrapper around a batch loader of images, labels,
+and predictions, that allows deepchecks to efficiently calculate different
+:doc:`checks </checks_gallery/vision>` on your data, by caching some of the information.
+
+Information about the supported task types and the required formats for each task is available
+:doc:`here </user-guide/vision/supported_tasks_and_formats>`.
+
```

Comment:
Would add here a file structure with links..
(Common Class Parameters,
How to Create a VIsionData...,
How to Add Model predictions...)

Reply:
@Nadav-Barak

Reply:
Added